src/components/Viewer/Viewer.js

```diff
@@ -9,6 +9,7 @@ import MenuItem from '@material-ui/core/MenuItem';
 import Card from '@material-ui/core/Card';
 import CardContent from '@material-ui/core/CardContent';
 import Typography from '@material-ui/core/Typography';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
You should not have to  do this, if you are getting this error you are probably not installing the package correctly. Perhaps try `npm install react-inlinesvg'. Maybe the dependency is a dev dependency.